cli/oci.go

```diff
@@ -40,30 +42,32 @@ var cgroupsDirPath string
 
 var procMountInfo = "/proc/self/mountinfo"
 
+var ctrsMapTreePath = "/var/run/kata-containers/containers-mapping"
```

Comment:
Please put it in the runtime root storage directory instead of a hard coded one.

Reply:
As agreed, this should be done in a follow up PR introducing the support of `kata-runtime --root`